### PR TITLE
Inits App Mesh with EKS walkthrough

### PR DESCRIPTION
**Pull request by mhausenblas**
*Description of changes:*

End-to-end docs for using App Mesh with EKS, based on inject/controller. Currently covers day 2 ops topic o11y with CloudWatch.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: walkthroughs/eks/base.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+$ kubectl -n appmesh-inject get po
+NAME                                  READY   STATUS    RESTARTS   AGE
+aws-app-mesh-inject-c6f55c565-xtnh7   1/1     Running   0          20s
+```
+
+Next, install the [AWS App Mesh Controller For Kubernetes](https://github.com/aws/aws-app-mesh-controller-for-k8s) along with the custom resources:
+
+```
+$ git clone https://github.com/aws/aws-app-mesh-controller-for-k8s.git
+$ cd aws-app-mesh-controller-for-k8
```
Comment (kiranmeduri) [inline walkthroughs/eks/base.md]: aws-app-mesh-controller-for-k8(s)

File: walkthroughs/eks/o11y-cloudwatch.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+![AppMesh console edit virtual node step 0](appmesh-log-0.png)
+
+Now, expand the 'Additional configuration' section and enter `/dev/stdout` in the 'HTTP access logs path' as shown in the following:
+
+![AppMesh console edit virtual node step 1](appmesh-log-1.png)
+
+Check by getting the virtual node configuration via the CLI:
+
+```
+$ aws appmesh describe-virtual-node \
```
Comment (kiranmeduri) [inline walkthroughs/eks/o11y-cloudwatch.md]: Output seems to be missing _Logging_ configuration.

File: walkthroughs/eks/base.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
+# App Mesh with EKS—Base Deployment
```
Comment (kiranmeduri) [inline walkthroughs/eks/base.md]: You may want to update the existing README that has pointers to EKS setup with old instructions.

Comment (mhausenblas) [reply walkthroughs/eks/base.md]: Not entirely sure which README you mean (this: https://github.com/aws/aws-app-mesh-examples/blob/master/README.md maybe?). If so, it currently points to the `examples` directory, yes and I could imagine adding a pointer to the `walkthroughs` directory as well. Unsure about how we want to handle this, maybe @abby-fuller has an opinion on this?

File: walkthroughs/eks/o11y-cloudwatch.md (hunk 2 above)
Comment (mhausenblas) [reply walkthroughs/eks/o11y-cloudwatch.md]: I double-checked: the logging part doesn't show up in the CLI output (and also not in the console). Removing this validation part for now until this is fixed.

File: walkthroughs/eks/base.md (hunk 3 above)
Comment (kiranmeduri) [reply walkthroughs/eks/base.md]: Under https://github.com/aws/aws-app-mesh-examples/blob/master/examples/README.md#infrastructure there is a pointer for EKS setup. That can point to this walkthrough.

Comment (mhausenblas) [reply walkthroughs/eks/base.md]: Got it, thanks @kiranmeduri.

**Comment by mhausenblas** [comment]
Thanks again for your review  @kiranmeduri and addressed all your comments.

File: examples/README.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ $ ./infrastructure/mesh.sh
 $ ./infrastructure/ecs-cluster.sh
 ```
 
-* Setup EKS Cluster (Optional if using EKS). Note that there are more steps to use Kubernetes cluster that are not covered here. Please follow [EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+* Setup EKS Cluster (Optional if using EKS). Note that there are more steps to use Kubernetes cluster that are not covered here. Please follow [EKS Getting Started Guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html). See also the [Walkthrough: App Mesh with EKS](../walkthroughs/eks/) for others options.
```
Comment (kiranmeduri) [inline examples/README.md]: nit:others -> other

Comment (mhausenblas) [reply examples/README.md]: Ha, eagle eye! Thanks and fixed in 533dbe3f34b1e16c52107c083384cd0786971159

**Pull request merged**